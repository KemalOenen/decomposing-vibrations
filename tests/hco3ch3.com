***,hco3ch3 CAME vibrational spectrum
memory,1000,m

orient,mass
geometry={
O1
C1,O1,B1
O2,C1,B2,O1,A1
H1,O2,B3,C1,A2,O1,0.0
O3,C1,B4,O1,A3,H1,180.0
C2,O3,B5,C1,A4,O1,0.0
H2,C2,B6,O3,A5,C1,180.0
H3,C2,B7,O3,A6,C1,D1
H4,C2,B7,O3,A6,C1,-D1
}
 
B1=                  1.20826386 ANG
B2=                  1.34632938 ANG
A1=                125.37105052 DEGREE
B3=                  0.96620055 ANG
A2=                104.93520844 DEGREE
B4=                  1.33375801 ANG
A3=                126.65441654 DEGREE
B5=                  1.43708653 ANG
A4=                113.41206325 DEGREE
B6=                  1.08375252 ANG
A5=                105.23462880 DEGREE
B7=                  1.08668081 ANG
A6=                110.40633463 DEGREE
D1=                 60.53271261 DEGREE
 
mass,iso
basis=vtz-f12
{hf
start,atden}
ccsd(t)-f12a
optg
{freq,symm=auto, print=0}
