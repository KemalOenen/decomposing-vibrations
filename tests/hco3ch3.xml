<?xml version="1.0"?>
<molpro xmlns="http://www.molpro.net/schema/molpro-output"
  xmlns:xsd="http://www.w3.org/1999/XMLSchema"
  xmlns:cml="http://www.xml-cml.org/schema"
  xmlns:stm="http://www.xml-cml.org/schema"
  xmlns:xhtml="http://www.w3.org/1999/xhtml">
 <job>
  <jobstep command="HF-SCF" commandset="SCFPRO">
   <cml:molecule>
    <cml:symmetry pointGroup="Cs">
     <cml:transform3 title="generator" id="Y">
       1  0  0  0  0 -1  0  0  0  0  1  0  0  0  0  1
     </cml:transform3>
    </cml:symmetry>
    <cml:atomArray>
     <cml:atom id="a1" elementType="O" x3="-1.30408551989187" y3="0.0" z3="0.417962188848277"/>
     <cml:atom id="a2" elementType="C" x3="-0.959889919823881E-01" y3="0.0"
       z3="0.438070262727998"/>
     <cml:atom id="a3" elementType="O" x3="0.664981619244618" y3="0.0" z3="1.54871265715565"/>
     <cml:atom id="a4" elementType="H" x3="0.355976961208979E-01" y3="0.0"
       z3="2.28180158773208"/>
     <cml:atom id="a5" elementType="O" x3="0.717944248750302" y3="0.0" z3="-0.61853898806888"/>
     <cml:atom id="a6" elementType="C" x3="0.216706927545954E-01" y3="0.0"
       z3="-1.87568688307906"/>
     <cml:atom id="a7" elementType="H" x3="0.79843205804779" y3="0.0" z3="-2.63143902895375"/>
     <cml:atom id="a8" elementType="H" x3="-0.600193842452052" y3="0.886729954740416"
       z3="-1.964398043823"/>
     <cml:atom id="a9" elementType="H" x3="-0.600193842452052" y3="-0.886729954740416"
       z3="-1.964398043823"/>
    </cml:atomArray>
    <cml:bondArray>
     <cml:bond atomRefs2="a1 a2"/>
     <cml:bond atomRefs2="a2 a3"/>
     <cml:bond atomRefs2="a3 a4"/>
     <cml:bond atomRefs2="a2 a5"/>
     <cml:bond atomRefs2="a5 a6"/>
     <cml:bond atomRefs2="a6 a7"/>
     <cml:bond atomRefs2="a6 a8"/>
     <cml:bond atomRefs2="a6 a9"/>
    </cml:bondArray>
   </cml:molecule>
   <property name="Energy" method="RHF" principal="true" stateSymmetry="1" stateNumber="1"
     value="-302.81644419275"/>
   <property name="One-electron energy" method="RHF" value="-781.095832604855"/>
   <property name="Two-electron energy" method="RHF" value="293.822313118835"/>
   <property name="Kinetic energy" method="RHF" value="302.413770008439"/>
   <property name="Nuclear energy" method="RHF" value="184.45707529327"/>
   <property name="Virial quotient" method="RHF" value="-1.00133153389245"/>
   <property name="Dipole moment" method="RHF" principal="true" stateSymmetry="1"
     stateNumber="1" value="0.129255662181732 0.0 -0.26164885087285"/>
   <time start="13:33:39" end="13:35:48" cpu="505.15" system="19.95" real="128.64"/>
   <storage units="megabyte" memory="0.0" sf="0.0" df="11516.6" eaf="0.0"/>
   <summary overall_method="HF/cc-pVTZ-F12"/>
  </jobstep>
  <jobstep command="CCSD(T)-F12A" commandset="CCSD">
   <property name="energy" method="Reference" stateSymmetry="1" stateNumber="1"
     value="-302.81644419275"/>
   <property name="" method="F12 singles correction" stateSymmetry="1" stateNumber="1"
     value="-0.286334269010835E-02"/>
   <property name="energy" method="New reference" stateSymmetry="1" stateNumber="1"
     value="-302.81930753544"/>
   <property name="singlet correction" method="F12a" stateSymmetry="1" stateNumber="1"
     value="-0.8455289320025E-01"/>
   <property name="triplet correction" method="F12a" stateSymmetry="1" stateNumber="1"
     value="-0.10432536314102E-01"/>
   <property name="total correction" method="F12a" stateSymmetry="1" stateNumber="1"
     value="-0.94985429514352E-01"/>
   <property name="singlet pair energy" method="CCSD-F12a" stateSymmetry="1" stateNumber="1"
     value="-0.789844207457111"/>
   <property name="triplet pair energy" method="CCSD-F12a" stateSymmetry="1" stateNumber="1"
     value="-0.371520312358813"/>
   <property name="correlation energy" method="CCSD-F12a" stateSymmetry="1" stateNumber="1"
     value="-1.16136466076619"/>
   <property name="contribution" method="Triples (T)" stateSymmetry="1" stateNumber="1"
     value="-0.458499315468936E-01"/>
   <property name="correlation energy" method="Total" stateSymmetry="1" stateNumber="1"
     value="-1.20721459231309"/>
   <property name="total energy" method="CCSD-F12a" stateSymmetry="1" stateNumber="1"
     value="-303.980672196206"/>
   <property name="energy" method="CCSD[T]-F12a" stateSymmetry="1" stateNumber="1"
     value="-304.028762172096"/>
   <property name="energy" method="CCSD-T-F12a" stateSymmetry="1" stateNumber="1"
     value="-304.025644478467"/>
   <property name="total energy" method="CCSD(T)-F12a" principal="true" stateSymmetry="1"
     stateNumber="1" value="-304.026522127753"/>
   <time start="13:35:48" end="13:47:16" cpu="3062.59" system="45.65" real="687.85"/>
   <storage units="megabyte" memory="0.0" sf="0.0" df="11516.6" eaf="0.0"/>
   <summary overall_method="CCSD(T)-F12A/cc-pVTZ-F12"/>
  </jobstep>
  <jobstep command="OPTG" commandset="OPT">
