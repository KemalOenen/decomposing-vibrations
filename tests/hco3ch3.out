
 Working directory              : /tmp/koe/molpro.D8PK6X2LL7/
 Global scratch directory       : /tmp/koe/molpro.D8PK6X2LL7/
 Wavefunction directory         : /home/koe/wfu/
 Main file repository           : /tmp/koe/molpro.D8PK6X2LL7/

 id        : liedlgroup

 Nodes     nprocs
 zh2-c724     1
 GA implementation: MPI file
 GA implementation (serial work in mppx): MPI file

 Using customized tuning parameters: mindgm=1; mindgv=20; mindgc=4; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  


 Variables initialized (1015), CPU time= 0.00 sec
 ***,hco3ch3 CAME vibrational spectrum
 memory,1000,m
 
 orient,mass
 geometry={
 O1
 C1,O1,B1
 O2,C1,B2,O1,A1
 H1,O2,B3,C1,A2,O1,0.0
 O3,C1,B4,O1,A3,H1,180.0
 C2,O3,B5,C1,A4,O1,0.0
 H2,C2,B6,O3,A5,C1,180.0
 H3,C2,B7,O3,A6,C1,D1
 H4,C2,B7,O3,A6,C1,-D1
 }
 
 B1=                  1.20826386 ANG
 B2=                  1.34632938 ANG
 A1=                125.37105052 DEGREE
 B3=                  0.96620055 ANG
 A2=                104.93520844 DEGREE
 B4=                  1.33375801 ANG
 A3=                126.65441654 DEGREE
 B5=                  1.43708653 ANG
 A4=                113.41206325 DEGREE
 B6=                  1.08375252 ANG
 A5=                105.23462880 DEGREE
 B7=                  1.08668081 ANG
 A6=                110.40633463 DEGREE
 D1=                 60.53271261 DEGREE
 
 mass,iso
 basis=vtz-f12
 {hf
 start,atden}
 ccsd(t)-f12a
 optg
 {freq,symm=auto, print=0}
 Commands initialized (834), CPU time= 0.01 sec, 672 directives.
 Default parameters read. Elapsed time= 0.08 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2022.2 linked Thu May  5 22:39:45 2022


 **********************************************************************************************************************************
 LABEL *   hco3ch3 CAME vibrational spectrum                                             
  (6 PROC) 64 bit serial version                                                         DATE: 26-Sep-22          TIME: 13:33:39  
 **********************************************************************************************************************************

 SHA1:             518bf5c5e315925380b88506c70af759aa1d1d96
 **********************************************************************************************************************************

 Memory per process:      1000 MW
 Total memory per node:   1000 MW

 GA preallocation disabled
 GA check disabled

 Variable memory set to 1000.0 MW

 ZCENTRE=MASS

 SETTING B1             =         1.20826386  ANG                             
 SETTING B2             =         1.34632938  ANG                             
 SETTING A1             =       125.37105052  DEGREE                          
 SETTING B3             =         0.96620055  ANG                             
 SETTING A2             =       104.93520844  DEGREE                          
 SETTING B4             =         1.33375801  ANG                             
 SETTING A3             =       126.65441654  DEGREE                          
 SETTING B5             =         1.43708653  ANG                             
 SETTING A4             =       113.41206325  DEGREE                          
 SETTING B6             =         1.08375252  ANG                             
 SETTING A5             =       105.23462880  DEGREE                          
 SETTING B7             =         1.08668081  ANG                             
 SETTING A6             =       110.40633463  DEGREE                          
 SETTING D1             =        60.53271261  DEGREE                          


 Using masses of most abundant isotopes

 **********************************************************************************************************************************
 SETTING BASIS          =    VTZ-F12


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry O      S cc-pVTZ-F12          selected for orbital group  1
 Library entry O      P cc-pVTZ-F12          selected for orbital group  1
 Library entry O      D cc-pVTZ-F12          selected for orbital group  1
 Library entry O      F cc-pVTZ-F12          selected for orbital group  1
 Library entry C      S cc-pVTZ-F12          selected for orbital group  2
 Library entry C      P cc-pVTZ-F12          selected for orbital group  2
 Library entry C      D cc-pVTZ-F12          selected for orbital group  2
 Library entry C      F cc-pVTZ-F12          selected for orbital group  2
 Library entry H      S cc-pVTZ-F12          selected for orbital group  4
 Library entry H      P cc-pVTZ-F12          selected for orbital group  4
 Library entry H      D cc-pVTZ-F12          selected for orbital group  4


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Asymmetric top
 Symmetry elements: Y
 Rotational constants:       4.4570786      3.2435983     11.0853145 GHz

 Point group  Cs  



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  O1      8.00   -2.464364476    0.000000000    0.789834067
   2  C1      6.00   -0.181392906    0.000000000    0.827832820
   3  O2      8.00    1.256633138    0.000000000    2.926642768
   4  H1      1.00    0.067269896    0.000000000    4.311980072
   5  O3      8.00    1.356718003    0.000000000   -1.168869285
   6  C2      6.00    0.040951674    0.000000000   -3.544534505
   7  H2      1.00    1.508817919    0.000000000   -4.972699078
   8  H3      1.00   -1.134201984    1.675676761   -3.712174303
   9  H4      1.00   -1.134201984   -1.675676761   -3.712174303

 Bond lengths in Bohr (Angstrom)

 1-2  2.283287782  2-3  2.544193802  2-5  2.520437355  3-4  1.825854421  5-6  2.715699959
     ( 1.208263860)     ( 1.346329380)     ( 1.333758010)     ( 0.966200550)     ( 1.437086530)

 6-7  2.047995450  6-8  2.053529116  6-9  2.053529116
     ( 1.083752520)     ( 1.086680810)     ( 1.086680810)

 Bond angles

  1-2-3  125.37105052   1-2-5  126.65441654   2-3-4  104.93520844   2-5-6  113.41206325

  3-2-5  107.97453294   5-6-7  105.23462880   5-6-8  110.40633463   5-6-9  110.40633463

  7-6-8  110.68501313   7-6-9  110.68501313   8-6-9  109.37243054

 NUCLEAR CHARGE:                   40
 NUMBER OF PRIMITIVE AOS:         444
 NUMBER OF SYMMETRY AOS:          395
 NUMBER OF CONTRACTIONS:          337   (  219A'  +  118A"  )
 NUMBER OF INNER CORE ORBITALS:     0   (    0A'  +    0A"  )
 NUMBER OF OUTER CORE ORBITALS:     5   (    5A'  +    0A"  )
 NUMBER OF VALENCE ORBITALS:       24   (   18A'  +    6A"  )


 NUCLEAR REPULSION ENERGY  184.45707529


 Eigenvalues of metric

         1 0.581E-04 0.982E-04 0.138E-03 0.171E-03 0.208E-03 0.237E-03 0.296E-03 0.362E-03
         2 0.289E-03 0.527E-03 0.561E-03 0.677E-03 0.682E-03 0.691E-03 0.169E-02 0.297E-02


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     3154.641 MB (compressed) written to integral file ( 42.7%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:  822234496.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:  26  SEGMENT LENGTH:   31996000      RECORD LENGTH: 524288

 Memory used in sort:      32.55 MW

 SORT1 READ   923025426. AND WROTE   736095372. INTEGRALS IN   2119 RECORDS. CPU TIME:     9.15 SEC, REAL TIME:    14.25 SEC
 SORT2 READ   736095372. AND WROTE   822234496. INTEGRALS IN  12034 RECORDS. CPU TIME:     9.44 SEC, REAL TIME:    12.28 SEC

 FILE SIZES:   FILE 1:  3187.8 MBYTE,  FILE 4:  8887.8 MBYTE,   TOTAL:  12075.6 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19     2827.51       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *       179.96    179.86
 REAL TIME  *        69.92 SEC
 DISK USED  *        11.25 GB
 **********************************************************************************************************************************


 Program * Restricted Hartree-Fock

 Orbital guess generated from atomic densities. Full valence occupancy:   23   6

 Initial occupancy:  16   4

 NELEC=   40   SYM=1   MS2= 0   THRE=1.0D-08   THRD=1.0D-06   THRG=1.0D-06  HFMA2=F  DIIS_START=2   DIIS_MAX=10   DIIS_INCORE=F

 Level shifts:    0.00 (CLOSED)    0.00 (OPEN)    0.30 (GAP_MIN)

 ITER           ETOT              DE          GRAD        DDIFF     DIIS  NEXP   TIME(IT)  TIME(TOT)  DIAG
   1     -302.69514233    -302.69514233     0.00D+00     0.50D-01     0     0      25.77     50.95    start
   2     -302.77193461      -0.07679228     0.64D-02     0.66D-02     1     0      24.91     75.86    diag
   3     -302.81465280      -0.04271818     0.42D-02     0.28D-02     2     0      24.88    100.74    diag
   4     -302.81614204      -0.00148924     0.59D-03     0.52D-03     3     0      24.96    125.70    diag
   5     -302.81640130      -0.00025926     0.26D-03     0.23D-03     4     0      24.78    150.48    diag
   6     -302.81643782      -0.00003652     0.72D-04     0.83D-04     5     0      25.02    175.50    diag
   7     -302.81644343      -0.00000562     0.24D-04     0.38D-04     6     0      24.93    200.43    diag
   8     -302.81644411      -0.00000067     0.81D-05     0.14D-04     7     0      24.78    225.21    fixocc
   9     -302.81644418      -0.00000007     0.29D-05     0.44D-05     8     0      24.83    250.04    diag
  10     -302.81644419      -0.00000001     0.10D-05     0.17D-05     9     0      25.16    275.20    diag/orth
  11     -302.81644419      -0.00000000     0.40D-06     0.49D-06     9     0      25.02    300.22    diag
  12     -302.81644419      -0.00000000     0.11D-06     0.13D-06     0     0      24.92    325.14    diag

 Final occupancy:  16   4

 !RHF STATE 1.1 Energy               -302.816444192750
  RHF One-electron energy            -781.095832604855
  RHF Two-electron energy             293.822313118835
  RHF Kinetic energy                  302.413770008439
  RHF Nuclear energy                  184.457075293270
  RHF Virial quotient                  -1.001331533892

 !RHF STATE 1.1 Dipole moment           0.12925566     0.00000000    -0.26164885
 Dipole moment /Debye                   0.32853512     0.00000000    -0.66504504

 Orbital energies:

           1.1          2.1          3.1          4.1          5.1          6.1          7.1          8.1          9.1         10.1
    -20.623024   -20.618727   -20.553281   -11.440740   -11.293795    -1.515478    -1.409416    -1.372743    -0.992062    -0.836876

          11.1         12.1         13.1         14.1         15.1         16.1         17.1         18.1
     -0.754174    -0.721490    -0.635037    -0.616626    -0.548031    -0.482917     0.046043     0.051195

           1.2          2.2          3.2          4.2          5.2          6.2
     -0.704756    -0.612959    -0.511559    -0.481899     0.064164     0.092576


 HOMO      4.2    -0.481899 =     -13.1131eV
 LUMO     17.1     0.046043 =       1.2529eV
 LUMO-HOMO         0.527942 =      14.3660eV

 Orbitals saved in record  2100.2


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19     2827.51       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              2       4        1.45       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL    HF-SCF       INT
 CPU TIMES  *       505.15    325.18    179.86
 REAL TIME  *       128.64 SEC
 DISK USED  *        11.25 GB
 **********************************************************************************************************************************


 PROGRAM * CCSD (Closed-shell coupled cluster)     Authors: C. Hampel, H.-J. Werner, 1991, M. Deegan, P.J. Knowles, 1992

                                  CCSD-F12 implementation by  H.-J. Werner, 2007

                   Density fitting integral evaluation by F.R. Manby, 2003,2007, G. Knizia, 2010


 Basis set VTZ-F12/JKFIT generated.      Number of basis functions:   704 
 Basis set CC-PVTZ-F12/OPTRI generated.  Number of basis functions:   539 
 Basis set VTZ-F12/MP2FIT generated.     Number of basis functions:   714 

 Convergence thresholds:  THRVAR = 1.00D-12  THRDEN = 1.00D-08

 CCSD(T)     terms to be evaluated (factor= 1.000)


 Number of core orbitals:           5 (   5   0 )
 Number of closed-shell orbitals:  15 (  11   4 )
 Number of external orbitals:     317 ( 203 114 )

 Molecular orbitals read from record     2100.2  Type=RHF/CANONICAL  

 MP2-F12 correlation treatment (H.-J. Werner, 2006)
 ==================================================

 Using MP2-F12 with ansatz 3C(FIX)

 Using projected zeroth-order Hamiltonian (+Z)

 FOCKRIB=T FOCKRIC=T FOCKRIP=T CABSP=T CABSA=T CABSK=T CABSF=T GBC=F EBC=F DMAT=T NOFIK=T NOPAO=1 SOLVE=-1  USEPAO=0
 EXCH_A= T EXCH_B= F EXCH_C= F EXCH_P= F

 Geminal basis:    OPTFULL  GEM_TYPE=SLATER  BETA=1.0  NGEM=6

 Optimizing Gaussian exponents for each gem_beta

 Geminal optimization for beta= 1.0000
 Weight function:   m=0, omega= 1.4646

 Augmented Hessian optimization of geminal fit. Trust ratio= 0.40000
 Convergence reached after   2 iterations. Final gradient= 9.01D-16, Step= 4.09D-06, Delta= 1.28D-09

 Alpha:                 0.19532     0.81920     2.85917     9.50073    35.69989   197.79328
 Coeff:                 0.27070     0.30552     0.18297     0.10986     0.06810     0.04224


 Symmetry turned off. Reference energy:       -302.816444192750

 All pairs explicitly correlated. Number of r12-pairs:          120

 Excluding core orbitals from MO domains

 AO(A)-basis ORBITAL           loaded. Number of functions:     337
 RI(R)-basis CC-PVTZ-F12/OPTRI loaded. Number of functions:     539
 DF-basis VTZ-F12/JKFIT        loaded. Number of functions:     704

 Screening thresholds:   THRAO=  1.00D-10  THRMO=  1.00D-10  THRPROD=  1.00D-10
                         THRSW=  1.00D-07  THROV=  1.00D-12  THRAOF12= 1.00D-08

 CPU time for Fock operators                     29.69 sec

 Construction of ABS:
 Smallest eigenvalue of S          3.10E-04  (threshold= 1.00E-08)
 Ratio eigmin/eigmax               3.61E-05  (threshold= 1.00E-09)
 Smallest eigenvalue of S kept     3.10E-04  (threshold= 3.10E-04, 0 functions deleted, 539 kept)

 Construction of CABS:
 Smallest eigenvalue of S          3.67E-07  (threshold= 1.00E-08)
 Ratio eigmin/eigmax               3.67E-07  (threshold= 1.00E-09)
 Smallest eigenvalue of S kept     3.67E-07  (threshold= 3.67E-07, 0 functions deleted, 539 kept)

 CPU time for CABS singles                        0.45 sec

 CABS-singles contribution of  -0.00286334 patched into reference energy.
 New reference energy        -302.81930754

 AO(A)-basis ORBITAL           loaded. Number of functions:     337
 RI(R)-basis CC-PVTZ-F12/OPTRI loaded. Number of functions:     539
 DF-basis VTZ-F12/MP2FIT       loaded. Number of functions:     714

 Screening thresholds:   THRAO=  1.00D-10  THRMO=  1.00D-10  THRPROD=  1.00D-10
                         THRSW=  1.00D-07  THROV=  1.00D-12  THRAOF12= 1.00D-08

 CPU time for 3-index integral evaluation        43.46 sec
 CPU time for first  half transformation          3.92 sec ( 8314.2 MFLOP/sec)
 CPU time for second half transformation          0.83 sec ( 1217.6 MFLOP/sec)
 CPU time for sorting                             0.17 sec
 CPU time for fitting                             1.47 sec ( 9738.1 MFLOP/sec)
 CPU time for tilde quantities                    1.20 sec (15969.4 MFLOP/sec)
 CPU time for assembly                           11.85 sec (26822.7 MFLOP/sec)
 CPU time for tranop_f12                         22.94 sec (18608.3 MFLOP/sec)
 CPU time for f12 integrals (total)             109.54 sec, Elapsed time:     31.91 sec
 CPU time for f12 matrices (total)                9.42 sec, Elapsed time:      1.80 sec

 Diagonal F12 approximation with fixed coefficients:  TSING= 0.500,  TTRIP= 0.250 (scaled by -1/beta)

 DF-MP2-F12 energy corrections:
 ==============================
 Approx.                                    Singlet             Triplet             Total
 DF-MP2-F12/3*C(DX,FIX)                -0.091631948601     -0.011417584101     -0.103049532701
 DF-MP2-F12/3*C(FIX)                   -0.085395777634     -0.010462149298     -0.095857926932
 DF-MP2-F12/3C(FIX)                    -0.085318390109     -0.010893831953     -0.096212222062

 DF-MP2-F12 correlation energies:
 ================================
 Approx.                                    Singlet             Triplet             Ecorr            Total Energy
 DF-MP2                                -0.666423247271     -0.406747666089     -1.073170913360   -303.892478448799
 DF-MP2-F12/3*C(DX,FIX)                -0.758055195871     -0.418165250190     -1.176220446061   -303.995527981501
 DF-MP2-F12/3*C(FIX)                   -0.751819024905     -0.417209815387     -1.169028840292   -303.988336375731
 DF-MP2-F12/3C(FIX)                    -0.751741637380     -0.417641498042     -1.169383135422   -303.988690670862

 SCS-DF-MP2 energies (F_SING= 1.20000  F_TRIP= 0.62222  F_PARALLEL= 0.33333):
 ============================================================================
 SCS-DF-MP2                            -1.052795333402   -303.872102868842
 SCS-DF-MP2-F12/3*C(DX,FIX)            -1.169857946275   -303.989165481714
 SCS-DF-MP2-F12/3*C(FIX)               -1.161780048348   -303.981087583788
 SCS-DF-MP2-F12/3C(FIX)                -1.161955785860   -303.981263321299

 Symmetry restored.
 Symmetry transformation completed.

 Number of N-1 electron functions:              15
 Number of N-2 electron functions:             120
 Number of singly external CSFs:              2689
 Number of doubly external CSFs:           5751916
 Total number of CSFs:                     5754606

 Length of J-op  integral file:               1.48 GB
 Length of K-op  integral file:               1.64 GB
 Length of 3-ext integral record:             0.00 MB

 Memory could be reduced to 295.82 Mwords without degradation in triples

 Integral transformation finished. Total CPU: 130.23 sec, npass=  1  Memory used:  96.37 MW

 Reference energy:                   -302.81930754

 Adding F12 terms to K(Cij), methodcc=4,  factor= 1.0

 ITER.      SQ.NORM     CORR.ENERGY   TOTAL ENERGY   ENERGY CHANGE        DEN1      VAR(S)    VAR(P)  DIIS     TIME  TIME/IT
   1      1.25535666    -1.06801686  -303.88732440     0.00525941     0.03537720  0.65D-02  0.52D-02  1  1   384.36    17.45
   2      1.27258738    -1.06615790  -303.88546544     0.00185896    -0.02326938  0.99D-04  0.66D-03  2  2   491.16    18.00
   3      1.27877761    -1.06651008  -303.88581761    -0.00035217    -0.00293151  0.14D-03  0.39D-04  3  3   597.63    18.16
   4      1.28223336    -1.06642017  -303.88572771     0.00008991    -0.00191012  0.11D-04  0.48D-05  4  4   703.55    18.21
   5      1.28332727    -1.06641083  -303.88571836     0.00000934    -0.00034600  0.18D-05  0.46D-06  5  5   811.33    18.31
   6      1.28362905    -1.06638471  -303.88569225     0.00002611    -0.00001369  0.13D-06  0.50D-07  6  6   918.79    18.37
   7      1.28368845    -1.06638009  -303.88568762     0.00000462    -0.00000945  0.12D-07  0.72D-08  6  1  1026.26    18.41
   8      1.28369120    -1.06637988  -303.88568741     0.00000021     0.00000221  0.28D-08  0.94D-09  6  3  1133.92    18.44
   9      1.28369797    -1.06637909  -303.88568663     0.00000079    -0.00000274  0.43D-09  0.13D-09  6  2  1241.85    18.48
  10      1.28369726    -1.06637923  -303.88568677    -0.00000014     0.00000056  0.86D-10  0.23D-10  6  5  1348.41    18.48
  11      1.28369782    -1.06637921  -303.88568675     0.00000002    -0.00000008  0.11D-10  0.45D-11  6  4  1455.62    18.50
  12      1.28369775    -1.06637922  -303.88568676    -0.00000001     0.00000002  0.17D-11  0.64D-12  6  6  1563.88    18.52
  13      1.28369772    -1.06637923  -303.88568677    -0.00000001     0.00000000  0.26D-12  0.13D-12  6  1  1671.78    18.54
  14      1.28369768    -1.06637923  -303.88568677     0.00000000    -0.00000000  0.47D-13  0.19D-13  6  3  1779.25    18.54

 Norm of t1 vector:      0.10716689      S-energy:    -0.00000014      T1 diagnostic:  0.01383519
                                                                       D1 diagnostic:  0.05435224
                                                                       D2 diagnostic:  0.15116722 (external, symmetry=2)


 Total CPU time for triples:   1282.67 sec


 RESULTS
 =======

  Reference energy                   -302.816444192750
  F12 singles correction               -0.002863342690

 F12 singles corrections added to reference energy

  New reference energy               -302.819307535440

  F12a singlet correction              -0.084552893200
  F12a triplet correction              -0.010432536314
  F12a total correction                -0.094985429514

 F12a corrections for ansatz F12/3C(FIX) added to CCSD energy

  CCSD-F12a singlet pair energy        -0.789844207457
  CCSD-F12a triplet pair energy        -0.371520312359
  CCSD-F12a correlation energy         -1.161364660766

  Triples (T) contribution             -0.045849931547
  Total correlation energy             -1.207214592313

  CCSD-F12a total energy             -303.980672196206
  CCSD[T]-F12a energy                -304.028762172096
  CCSD-T-F12a energy                 -304.025644478467
 !CCSD(T)-F12a total energy          -304.026522127753

 Timing summary (sec):

 STEP                 CPU(USER)    SYS     CPU(TOT)    WALL
 Transformation        131.14      2.86    134.00     28.26
 CCSD iterations      1498.48     15.34   1513.82    259.89
 Triples              1282.67     21.33   1304.00    344.02
 MP2-F12               150.27      5.65    155.92     55.16

 Program statistics:

 Available memory in ccsd:               999998484
 Min. memory needed in ccsd:              14231059
 Max. memory used in ccsd:                19502353
 Max. memory used in cckext:              20158159 (14 integral passes)



 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      20     3832.49       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700     1380   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER     JKOP   

              2       6        2.15       700     1000      520     2100     7360     7350   
                                         GEOM     BASIS   MCVARS     RHF    F12ABS    EF12   

 PROGRAMS   *        TOTAL   CCSD(T)    HF-SCF       INT
 CPU TIMES  *      3567.74   3062.59    325.18    179.86
 REAL TIME  *       816.49 SEC
 DISK USED  *        11.25 GB
 **********************************************************************************************************************************
